frame/assets/src/lib.rs

```diff
@@ -355,13 +355,19 @@ pub mod pallet {
 		match origin.into() {
 			Ok(frame_system::RawOrigin::Signed(account)) => {
 				match T::GovernanceRegistry::get(asset_id) {
-					Ok(SignedRawOrigin::Root) => Ok(()),
+					Ok(SignedRawOrigin::Root) => Ok(()), /* ISSUE: it says if
+					                                       * (call_origin.is_signed &&
+					                                       * asst_owner.is_root) then allow
+					                                       * mint/burn -> anybody can mint and
+					                                       * burn PICA? */
```

Comment:
Well this says:

`if root`, or `if caller == admin_origin`

Reply:
on line 351 it tells - of root or council => allow, else do next check

on line 357 it tells if signed => on line 359 it tells if asset owner is root -> then allow. 

so anybody signed can modify root owned asset.

line 364 tells if signed and if owner of asset is account then pass <- the only correct statement here (but it precludes collective to be owner)